lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/evaluator/EvaluatorState.java

```diff
@@ -31,7 +31,7 @@
   ALLOCATED,  // initial state
   SUBMITTED,  // client called AllocatedEvaluator.submitTask() and we're waiting for first contact
   RUNNING,    // first contact received, all communication channels established, Evaluator sent to client.
-  // TODO[JIRA REEF-832]: Add CLOSING state
+  CLOSING,    // evaluator is asked shutdown, but not closed yet.
```

Comment:
The wording is not quite clear. I would say something like *"Evaluator is in the process of shutting down"*, maybe?